Added "wheel" event to the carousel ♿

EthanThatOneKid opened the pull request:
The carousel now has a new event attached that waits for the user to use their scroll wheel to scroll over the event carousel. When the user scrolls down while hovering over the carousel, the carousel will move to the right.

EthanThatOneKid committed:
Update event-carousel.svelte

Added "wheel" event to the carousel.

EthanThatOneKid committed:
Replaced `onMount` with `on:` 🧠

For some reason, I completely forgot about the `on:` element directive in Svelte. Using this syntax, I can eliminate the need of `onMount` in order to subscribe event listeners to the DOM.

diamondburned commented on src/components/events/event-carousel.svelte:
```diff
@@ -29,43 +28,38 @@
+    <div
+      bind:this="{carouselButtonRight}"
+      class="carousel-button right"
+      on:click="{scrollLeft}"
```
This should probably be `scrollRight`

EthanThatOneKid committed:
Fixed typo "left" -> "right"

Resolving typo addressed in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/113#discussion_r628497348>.
